VersionControlUI/CMakeLists.txt

```diff
@@ -8,8 +8,10 @@ add_library(VersionControlUI SHARED
 	src/VersionControlUIException.h
 	src/versioncontrolui_api.h
 	src/VersionControlUIPlugin.h
+	src/DiffManager.h
 	src/VersionControlUIException.cpp
 	src/VersionControlUIPlugin.cpp
+	src/DiffManager.cpp
 	test/SimpleTest.cpp
 )
-envision_plugin(VersionControlUI InteractionBase)
+envision_plugin(VersionControlUI InteractionBase oomodel)
```

Comment:
We talked about keeping the VersionControl UI at the generic level. Why did you introduce a dependency on oomodel?